lib/archethic_web/controllers/api/web_hosting_controller.ex

```diff
@@ -102,6 +108,11 @@ defmodule ArchethicWeb.API.WebHostingController do
     end
   end
 
+  def published?(%ReferenceTransaction{json_content: %{"publicationStatus" => "UNPUBLISHED"}}),
```

Comment:
But ideally it should be part of the struct ReferenceTransaction to have function like `publication_status`